README.md

```
@@ -8,21 +8,162 @@
+
+```markdown
+## Installation
+
+### Set Up the Backend
+
+1. **Navigate to the Backend Directory**:
+   ```bash
+   cd backend
+   ```
+
+2. **Restore Dependencies**:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)